### DISCUSSION TITLE
docs: Comparison with other ORM / query builder / SQL libraries

### DISCUSSION THREAD

**Discussion by RohanTalip** [mikro-orm/mikro-orm#5532] [Ideas]
Hi @B4nan,

My company is currently using TypeORM but we're looking into alternatives, for various reasons.

I've been evaluating MikroORM against TypeORM and some other ORM / query builder / SQL libraries, such as Prisma, Sequelize, and Drizzle to name a few.  MikroORM seems to have a lot of similarities with TypeORM, at least with regards to the syntax used to define entities, but obviously there will be other differences.

Would you mind putting up some documentation that compares MikroORM against these libraries?  I'm particularly interested in comparisons against TypeORM and Drizzle at the moment, but comparisons against other libraries would also be interesting.

Performance comparisons like https://orm.drizzle.team/benchmarks would be useful.  I have seen https://mikro-orm.io/blog/mikro-orm-4-1-released but the referenced https://github.com/mikro-orm/benchmark doesn't exist any more.  It looks like you cloned https://github.com/emanuelcasco/typescript-orm-benchmark though, perhaps to contribute to it?

I'm curious how much the Unit of Work and Identity Map features of MikroORM benefit performance, compared to TypeORM and Drizzle.

Documentation on how to migrate from other ORMs would be useful too, e.g. https://orm.drizzle.team/learn/migrate/migrate-from-typeorm (and less important to me, but maybe interesting to other people: https://orm.drizzle.team/learn/migrate/migrate-from-sequelize and https://orm.drizzle.team/learn/migrate/migrate-from-prisma )

Thanks.

**Comment by thipokch** [comment]
@RohanTalip See https://github.com/drizzle-team/drizzle-northwind-benchmarks
